src/fu-util.c

```diff
@@ -4247,6 +4247,109 @@ fu_util_get_bios_setting(FuUtilPrivate *priv, gchar **values, GError **error)
 	return TRUE;
 }
 
+static gboolean
+fu_util_auto_repair(FuUtilPrivate *priv, gchar **values, GError **error)
+{
+	gchar *appstream_id = NULL;
+	g_autofree gchar *action = NULL;
+	g_autoptr(GList) fu_repair_list = NULL;
+	gint ret = 0;
+
+	for (guint i = 0; values[i] != NULL; i++) {
+		fu_console_print(priv->console, "%s: %s", _("Reparing"), values[i]);
+
+		if (i == 0) {
+			appstream_id = values[i];
+		} else if (i == 1) {
+			if (!g_strcmp0(values[i], "do")) {
+				action = g_strdup_printf("do");
+			} else if (!g_strcmp0(values[i], "undo")) {
+				action = g_strdup_printf("undo");
+			} else {
+				g_set_error_literal(error,
+						    FWUPD_ERROR,
+						    FWUPD_ERROR_INVALID_ARGS,
+						    /* TRANSLATOR: This is the error message for
+						     * incorrect parameter */
+						    _("Invalid arguments, expected do or undo"));
+				return FALSE;
+			}
+			break;
+		}
+	}
+
+	if (!appstream_id) {
+		g_set_error_literal(error,
+				    FWUPD_ERROR,
+				    FWUPD_ERROR_INVALID_ARGS,
+				    /* TRANSLATOR: This is the error message for
+				     * incorrect parameter */
+				    _("Invalid arguments, expected an AppStream ID"));
+		return FALSE;
+	}
+
+	if (!action)
+		action = g_strdup_printf("do");
+
+	ret = fwupd_client_repair(priv->client, appstream_id, action, priv->cancellable, error);
+	if (!ret)
+		return FALSE;
+
+	fu_console_print_full(priv->console,
+			      FU_CONSOLE_PRINT_FLAG_NONE,
+			      "%s\n",
+			      _("Repair Success"));
+
+	return TRUE;
+}
+
+static void
+title_print_padding(const gchar *title, GString *dst_string, gsize maxlen)
+{
+	gsize title_len;
+	gsize maxpad = maxlen;
+
+	if (maxlen == 0)
+		maxpad = 50;
+
+	if (title == NULL || dst_string == NULL)
+		return;
+	g_string_append_printf(dst_string, "%s", title);
+
+	title_len = g_utf8_strlen(title, -1) + 1;
+	for (gsize i = title_len; i < maxpad; i++)
+		g_string_append(dst_string, " ");
+}
+
+static void
+fu_util_repair_list(FuUtilPrivate *priv, GError **error)
+{
+	g_autoptr(GPtrArray) attrs = NULL;
+	g_autoptr(GString) repair_msg = g_string_new(NULL);
+	FuConsole *console = priv->console;
+
+	attrs = fwupd_client_get_host_security_attrs(priv->client, priv->cancellable, error);
+	if (attrs == NULL)
+		return;
+
+	for (guint j = 0; j < attrs->len; j++) {
+		FwupdSecurityAttr *attr = g_ptr_array_index(attrs, j);
+
+		title_print_padding(fwupd_security_attr_get_appstream_id(attr), repair_msg, 40);
+		g_string_append_printf(repair_msg, "%s\n", fwupd_security_attr_get_name(attr));
+	}
+
+	fu_console_print_full(console, FU_CONSOLE_PRINT_FLAG_NONE, "%s\n", repair_msg->str);
+}
+
+static gboolean
+fu_util_repair_list_items(FuUtilPrivate *priv, gchar **values, GError **error)
```

Comment:
This and all other functions should s/repair/security/ no?

Reply:
OKay. I'll replace them.